src/scripts/main.js

```diff
@@ -1,3 +1,90 @@
 'use strict';
 
-// write code here
+// section with all variables
+const container = document.querySelector('.container');
+const tBody = document.querySelector('tbody');
+const buttonAddRow = document.querySelector('.append-row');
+const buttonRemoveRow = document.querySelector('.remove-row');
+const buttonAddColumn = document.querySelector('.append-column');
+const buttonRemoveColumn = document.querySelector('.remove-column');
+const minNumOfRowsAndColLimit = 2;
+const maxNumOfRowsAndColLimit = 9;
+
+const rows = tBody.rows[0];
+
+container.addEventListener('click', function tableSizeManipulation(action) {
+  const target = action.target;
+
+  let numberOfRows = tBody.rows.length;
+  let numberOfColumns = tBody.rows[0].children.length;
+
+  // append rows
+  if (target.textContent === '+'
```

Comment:
```suggestion
  if (target === buttonAddColumn)
```
У тебя в этих переменных уже сохранены кнопки, и в target попадет одна из них